Service/Movie.cs

```diff
@@ -0,0 +1,86 @@
+﻿using RestSharp;
+using RestSharp.Serialization.Json;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Service
+{
+    public class Movie
+    {
+        private string _ApiMovieUrl = Util.Constantes.API_MOVIE_URL;
+        private string _ApiMoveiKey = Util.Constantes.API_MOVIE_KEY;
+        private string _ApiMoveiPath = Util.Constantes.API_MOVIE_PATHIMAGE;
+        private string _language = "pt-BR";
+
+        public async Task<Models.MovieUpComing> ProximosLancamentos(int page)
+        {
+            using (var cancellationTokenSource = new CancellationTokenSource())
```

Comment:
Por qual motivo esse _using_ está sendo utilizado?

Reply:
está sendo usado para fechar toda a conexão CancellationTokenSource, pois depois que é feita toda a chamada ao final teria que chamar Cancel e Dispose para fechar.
Com o using, isto já é feito automático.

Reply:
Para conexões HTTP sugiro sempre essa opção: https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests